brd-ios/breadwallet/src/Environment.swift

```diff
@@ -65,7 +65,7 @@ struct E {
         #if IS_EXTENSION_ENVIRONMENT
         return false
         #else
-        let bounds = UIApplication.shared.keyWindow?.bounds
+        let bounds = UIApplication.shared.windows.last?.bounds
```

Comment:
i think there is a better approach 

https://stackoverflow.com/questions/57134259/how-to-resolve-keywindow-was-deprecated-in-ios-13-0

Reply:
updated!